carbon-design-system/carbon
# Update ref handling and adjust label component indentation

- Updated the ref handling condition from `Object(ref) === ref` to `'current' in ref` for improved clarity and accuracy. This change simplifies the check for object refs and aligns with common React practices.
- Adjusted the indentation of the label component for consistency and readability. This helps maintain a clean and uniform code style throughout the component.

Closes #<issue_number>

#### Changelog

**New**

- N/A

**Changed**

- Ref handling condition updated from `Object(ref) === ref` to `'current' in ref` for improved clarity and alignment with React practices.
- Indentation of the `<label>` component adjusted for consistency and readability.

**Removed**

- N/A

#### Testing / Reviewing

To verify these changes:
1. Review the updated ref handling logic in the `Checkbox` component to ensure it properly distinguishes between function and object refs.
2. Verify that the modifications to the `<label>` start tag are correctly applied, ensuring there are no syntax errors or unexpected issues, and confirm that the tag displays properly.
